health_check.sh

```diff
@@ -15,10 +17,20 @@ check_site() {
         else
             message="🔴 **경고:** 사이트가 정상 응답을 못 하고 있어요! (status_code=$response) ($current_time)"
         fi
+
+        message="$message\n관련 액션 바로가기: $GITHUB_ACTION_URL"
         
```

Comment:
JSON으로 넘어가는 과정에서 개행 문자 깨짐이 발생할 수 있다고 해서 Raw 문자로 이스케이핑해서 $message를 {content: $message} 형태의 JSON payload로 만들어주는 코드입니다. GPT 의견이라서 틀린 말 같으면 무시하셔도 될 것 같습니다.

```suggestion
jq -Rn --arg content "$message" '{content: $content}'
```

Reply:
해당 부분 저도 GPT한테 들어서 넣었다가 ㅋㅋ 이스케이핑할 만한 문자가 없을 거 같아 다시 뺐습니다.
추후 이스케이핑 필요한 문자열이 나오면 적용하고, 현재는 스킵하겠습니다!

Reply:
리다이렉트 관련하여 주신 리뷰의 경우, curl에 리다이렉션을 따라가는 옵션([`-L`](https://curl.se/docs/manpage.html#-L))을 주는 것으로 수정했습니다! https://github.com/khu-khlug/sight-health-check/pull/2/commits/dbaafb9bb46280cacf542ec873bfd8261f4c1206
좋은 제안 감사해요~!